app/models/miq_request_task/state_machine.rb

```diff
@@ -70,14 +70,14 @@ def prematurely_finished?
     false
   end
 
-  def requeue_phase
+  def requeue_phase(delay = 10)
```

Comment:
Prefer 
```suggestion
  def requeue_phase(delay = 10.seconds)
```

Then below you just need `delay.from_now.utc`